src/message_interface/response.rs

```diff
@@ -57,6 +57,8 @@ pub enum Response {
     /// [`GetOutputsWithAdditionalUnlockConditions`](crate::message_interface::AccountMethod::
     /// GetOutputsWithAdditionalUnlockConditions)
     OutputIds(Vec<OutputId>),
+    /// Response for [`GetFoundryOutput`](crate::message_interface::AccountMethod::GetFoundryOutput)
+    FoundryOutput(OutputDto),
```

Comment:
@Tuditi would be another breaking change, but what do you think about merging `BuiltOutput(OutputDto),` and `FoundryOutput(OutputDto),` and just name it `OutputDto(OutputDto),`? Or would you rather want to keep them separated?

Reply:
Is this a breaking change, due to the interface change of `NativeTokenOptions`?

As far as I can see this would work. If `FoundryOutput` would contain the signature, blockId, transactionId of the `alias` address that created it, I would keep them separate. Why are these not included in the response of `GetFoundryOutput`?

I couldn't test the response, due to `minting failed Alias output creation took too long`.

Reply:
Why are you interested in the alias address and other stuff? Don't you just want to get the metadata?

Reply:
It could make sense if we want to display the token creation date? Not sure if this is necessary, but I just wanted to clarify it for myself as well. 

That being said, I think your suggestion makes sense, since the objects are exactly the same!

Reply:
Well I don't know what you really need :trollface:
So far the discussion was only about the native token metadata, which will be in the foundry output if the creator follows the standard

Reply:
All good then, I agree that we should merge the objects.

Reply:
@oddgrd we discussed a little bit more in private, please merge the responses to a single one and use it

Reply:
Alright, will do!